### DISCUSSION TITLE
Which charting library should we be using?

### DISCUSSION THREAD

**Discussion by dggsax** [microsoft/fluentui#18245] [Q&A] [answered]
I notice that there are two similarly-named charting packages:
 - @fluentui/react-charts - Looks like it's being produced from https://github.com/OfficeDev/microsoft-data-visualization-library.
 - @fluentui/react-charting - The one produced from this repository.

These are some differences between the two libraries, such as the react-charts one having [a website with documentation](https://officedev.github.io/microsoft-data-visualization-library/?path=/story/charts-line--default), but then the react-charting library having components like the Heatmap chart and doing things like `d3-scale` for data provided to charts to that labels don't have to be explicitly defined for time-series data.

We are about to begin investing some resources to producing more charts for an internal tool that we've built using FluentUI, so I was hoping to get some answers on which package we should be using.

Thank you very much in advance 😄

**Comment by harberg** [comment]
I am wondering as well which charting library we should be using. I found this one https://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/charting/dist/index.html#/ which seems to be the closest to what is in the @fluentui/react-charting code base.

Is there a link to updated examples from @fluentui/react-charting?

**Comment by khmakoto** [comment]
@Raghurk FYI

**Comment by micahgodbolt** [comment, marked as answer]
@fluentui/react-charting is the current project that Raghu and team are supporting.
